test/greet-test.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const greet = require('../lib/greet.js');
+const expect = require('chai').expect;
+var name = process.argv[2];
+
+describe('Greet', function() {
+  describe('#greet', function() {
+    it('should return "hello <name>"', function() {
+      var result = greet.greet(process.argv[2]);
```

Comment:
Similar comment to before, the name variable is already set to process.arv[2] so just another case of consistency here. Just pass name to greet.greet( )